thorlcr/slave/slave.cpp

```diff
@@ -83,16 +82,13 @@ void ProcessSlaveActivity::threadmain()
 #ifdef TIME_ACTIVITIES
         if (timeActivities)
         {
-            {
-                SpinBlock b(cycleLock);
-                lastCycles = get_cycles_now(); // serializeStats will reset
-            }
+            lastCycles.store(get_cycles_now()); // serializeStats will reset
```

Comment:
any reason this isn't lastCycles = get_cycles_now(); ?

Reply:
no - only to emphasise it is an atomic operation paired with the exchange below.  I can change if you prefer.

Reply:
not that fussed, but seems a little inconsistent if done here and not elsewhere.
Personally I'd make it lastCycles = get_cycles_now();